prometheus_juju_exporter/cli.py

```diff
@@ -26,7 +28,7 @@ def main(args=None):
         "-d", "--debug", dest="debug", action="store_true", help="Print debug log"
     )
 
-    parser, unknown = cli.parse_known_args(args)
+    parser, unknown = cli.parse_known_args(args)  # pylint: disable=W0612
```

Comment:
parser, _ = cli.parse_known_args(args)

https://stackoverflow.com/questions/3802410/is-it-possible-to-unpack-a-tuple-in-python-without-creating-unwanted-variables

Reply:
Changed. Thanks for the suggestions